pkg/webhookconfig/registration.go

```diff
@@ -131,7 +128,7 @@ func (wrc *Register) Register() error {
 	if wrc.serverIP != "" {
 		logger.Info("Registering webhook", "url", fmt.Sprintf("https://%s", wrc.serverIP))
 	}
-	if !wrc.debug {
+	if wrc.serverIP != "" {
```

Comment:
We already have a similar condition check at line 128, can be merge in single condition block

Reply:
done